e2e/homepage.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { expect, test } from '@playwright/test'
+import { testAuth } from './fixtures/authenticatedUserPages'
+import { waitForInitialPageLoad } from './helpers/utils'
+
+test.describe('Homepage', () => {
+  test('should show Log In To Synapse button when logged out', async ({
```

Comment:
Demonstrate unauthenticated and authenticated tests in the same file, although `testAuth` could be used for both unauthenticated and authenticated tests since only the fixtures needed for a particular test are setup for that test.

Reply:
Just to make sure I understand, you mean that we could just use the `page` fixture from `testAuth`  for the unauthenticated tests?

Reply:
Yeah, exactly -- since `page` is a default test fixture, and we haven't overridden it in `testAuth`, we could still use `page` from `testAuth` in unauthenticated tests. 

Also, since the worker fixtures are setup lazily for the first test that needs them, the unauthenticated tests wouldn't have to wait for the worker fixtures to be setup before running. 

I was mostly using this to organize tests (probably as a holdover from creating separate `loggedin`/`loggedout` test files), but if not's useful/helpful, we don't have to separate tests this way!